src/Samples/SubModelSeq/Program.fs

```diff
@@ -248,11 +248,11 @@ module Bindings =
     "AddCounter" |> Binding.cmd(fun m -> AddChild m.DummyRoot.Data.Id)
   ]
 
+let mainVm = ViewModel.designInstance (App.init ()) (Bindings.rootBindings ())
```

Comment:
Any reason this is called `mainVm` and not `vm` like the others?

Reply:
My intention was to use the name `vm` when there is a single XAML file because in that case there is no ambiguity.  This sample has two XAML files, which are named `MainWindow` and `Counter`, and this is the view model for `MainWindow`.  The fact that is it not easy to create a view model for the XAML file `Counter` (via bindings that are also used to define the parent bindings) is precisely what draft PR #244 is about.